cmd/lncli/cmd_macaroon.go

```diff
@@ -212,3 +214,59 @@ func listMacaroonIDs(ctx *cli.Context) error {
 	printRespJSON(resp)
 	return nil
 }
+
+var deleteMacaroonIDCommand = cli.Command{
+	Name:      "deletemacaroonid",
+	Category:  "Macaroons",
+	Usage:     "Delete a specific macaroon ID.",
+	ArgsUsage: "root_key_id",
+	Description: `
+	Remove a macaroon ID using the specified root key ID. For example:
+
+	lncli deletemacaroonid 1
+
+	Note that the default root key ID 0 cannot be deleted.
+	`,
+	Action: actionDecorator(deleteMacaroonID),
+}
+
+func deleteMacaroonID(ctx *cli.Context) error {
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+
+	// Show command help if no arguments.
+	if ctx.NArg() == 0 {
+		return cli.ShowCommandHelp(ctx, "deletemacaroonid")
+	}
+	// Validate args length. Only one argument is allowed.
```

Comment:
nit: missing new line above. Perhaps just printing the help if `ctx.NArg() != 1` is enough here.

Reply:
fixed